Build failure: openvdb (template error on darwin)

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

build openvdb

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console
/tmp/nix-build-openvdb-12.0.0.drv-0/source/openvdb/openvdb/../openvdb/tree/NodeManager.h:331:31: error: a template argument list
 is expected after a name prefixed by the template keyword [-Wmissing-template-arg-list-after-template-kw]
  331 |                 OpT::template eval(mNodeOp, it);
      |                               ^
/tmp/nix-build-openvdb-12.0.0.drv-0/source/openvdb/openvdb/../openvdb/tree/NodeManager.h:351:31: error: a template argument list
 is expected after a name prefixed by the template keyword [-Wmissing-template-arg-list-after-template-kw]
  351 |                 OpT::template eval(mNodeOp, it);
      |                               ^
/tmp/nix-build-openvdb-12.0.0.drv-0/source/openvdb/openvdb/../openvdb/tree/NodeManager.h:376:31: error: a template argument list is expected after a name prefixed by the template keyword [-Wmissing-template-arg-list-after-template-kw]
  376 |                 OpT::template eval(*mNodeOp, it);
      |                               ^
```

### Additional context

_No response_

### System metadata

- system: `"aarch64-darwin"`
 - host os: `Darwin 24.4.0, macOS 15.4`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.1`
 - nixpkgs: `/nix/store/zd0x43g1jcav0xy0k6q9pdhy3bvif77n-source`

### Notify maintainers

@guibou 


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/libraries/openvdb/default.nix
+++ b/pkgs/development/libraries/openvdb/default.nix
@@ -12,7 +12,7 @@
 
 stdenv.mkDerivation rec {
   pname = "openvdb";
-  version = "12.0.0";
+  version = "12.0.1";
 
   outputs = [
     "out"
@@ -22,8 +22,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "AcademySoftwareFoundation";
     repo = "openvdb";
-    rev = "v${version}";
-    sha256 = "sha256-S2uvzDCrTxAmvUMJr5PChcYTqhIHvRZbOfQLtUvzypI=";
+    tag = "v${version}";
+    hash = "sha256-ofVhwULBDzjA+bfhkW12tgTMnFB/Mku2P2jDm74rutY=";
   };
 
   nativeBuildInputs = [ cmake ];

